MAUI/Radial-Menu/Populating-Items.md

```diff
@@ -0,0 +1,723 @@
+---
+layout: post
+title: Populating Items in MAUI Radial Menu control | Syncfusion
+description: Learn here all about Populating Items support in Syncfusion MAUI Radial Menu (SfRadialMenu) control and more.
+platform: MAUI
+control: RadialMenu (SfRadialMenu)
+documentation: ug
+---
+
+# Populating Items in MAUI Radial Menu (SfRadialMenu)
+
+This section explains the ways about populating items through radial menu item and item source with item template.
+
+## Through radial menu items
+
+By passing a collection of [`SfRadialMenuItem`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.RadialMenu.SfRadialMenuItem.html), you can get the view of [`SfRadialMenu`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.RadialMenu.SfRadioButton.html) control. The radial menu item class provides various options to customize the items by giving custom views, font icons, and images. You can add radial menu items by hierarchy.
+
+### Text
+
+The following code snippet demonstrates how to Create a RadialMenu control with Items having the given text.
```

Comment:
The following code sample demonstrates how to Create a RadialMenu control with Items having the given text.